cg/services/orders/order_lims_service/order_lims_service.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from cg.apps.lims import LimsAPI
+from cg.models.lims.sample import LimsSample
+from cg.models.orders.order import OrderIn
+from cg.models.orders.samples import OrderInSample
+
+LOG = logging.getLogger(__name__)
+
+
+class OrderLimsService:
+
+    def __init__(self, lims_api: LimsAPI):
+        self.lims_api = lims_api
+
+    @staticmethod
+    def _build_lims_sample(customer: str, samples: list[OrderInSample]) -> list[LimsSample]:
+        """Convert order input to lims interface input."""
+        samples_lims = []
+        for sample in samples:
+            dict_sample = sample.__dict__
```

Comment:
This conversion to a dict seems weird to me. Is there a reason for it?

Reply:
Legacy code, this refactoring is not mean to address to code at that level at the moment. For now this refactoring is to inject dependencies and make future reworking easier.

Reply:
Expected as much. Some comments on the PR might just be ranting about stuff that's out of scope...